docs/pipelines/get-started/what-is-azure-pipelines.md

```diff
@@ -15,9 +15,9 @@ Azure Pipelines automatically builds and tests code projects to make them availa
 It works with just about any language or project type. Azure Pipelines combines continuous integration (CI) and continuous 
 delivery (CD) to constantly and consistently test and build your code and ship it to any target. 
 
-Continuous Integration (CI) is the practice used by development teams to automate the merging and testing of code.  Implementing CI helps to catch bugs early in the development cycle, which makes them less expensive to fix.  Automated tests execute as part of the CI process to ensure quality.  Artifacts are produced from CI systems and fed to release processes to drive frequent deployments. The Build service in TFS helps you set up and manage CI for your applications.
+Continuous Integration (CI) is the practice used by development teams to automate the merging and testing of code.  Implementing CI helps to catch bugs early in the development cycle, which makes them less expensive to fix.  Automated tests execute as part of the CI process to ensure quality.  Artifacts are produced from CI systems and fed to release processes to drive frequent deployments. The Build service in Azure DevOps (ADO) Server helps you set up and manage CI for your applications.
```

Comment:
```suggestion
Continuous Integration (CI) is the practice used by development teams to automate the merging and testing of code.  Implementing CI helps to catch bugs early in the development cycle, which makes them less expensive to fix.  Automated tests execute as part of the CI process to ensure quality.  Artifacts are produced from CI systems and fed to release processes to drive frequent deployments. The Build service in Azure DevOps Server helps you set up and manage CI for your applications.
```
Removing acronym, per the [style guide](https://styleguides.azurewebsites.net/Styleguide/Read?id=2696&topicid=44177)